lib/workers/repository/init/inherited.spec.ts

```diff
@@ -0,0 +1,91 @@
+import type { RenovateConfig } from '../../../config/types';
+import {
+  CONFIG_INHERIT_NOT_FOUND,
+  CONFIG_INHERIT_PARSE_ERROR,
+  CONFIG_VALIDATION,
+} from '../../../constants/error-messages';
+import { logger } from '../../../logger';
+import { platform } from '../../../modules/platform';
+import { mergeInheritedConfig } from './inherited';
+
+jest.mock('../../../modules/platform');
```

Comment:
isn't this already mocked globally?
```suggestion
```

https://github.com/renovatebot/renovate/blob/25ae92d631d34507c194a2fd4353069bc03bb528/test/setup.ts#L6